src/toil_rnaseq_sc/rnaseq_sc_cgl_pipeline.py

```diff
@@ -79,19 +81,20 @@ def run_single_cell(job, config, sample):
     with open(os.path.join(work_dir, "config.json"), 'w') as config_file:
         config_file.write(build_patcherlab_config(config))
     # Get Kallisto index
-    download_url(job, url=config.kallisto_index, name='kallisto_index.idx', work_dir=work_dir)
+    download_url(url=config.kallisto_index, name='kallisto_index.idx', work_dir=work_dir)
```

Comment:
`job` needs to be here.  I mislabelled the toil-lib dependency to 1.1.3 which doesn't have the latest refactoring in it.  Go ahead and change the `toil-lib` dependency in **version.py** to `toil-lib==1.2.0a1.dev139` for now.  Once Bd2kgenomics/toil-lib#78 is merged, I can release a new stable version which we'll depend on.